tests/sku-init/sku-init.test.ts

```diff
@@ -61,13 +61,6 @@ describe('sku init', () => {
     );
     const packageJson = JSON.parse(contents);
 
-    // `workspace:` prefix comes from the default `saveWorkspaceProtocol` value.
-    // See https://pnpm.io/settings#saveworkspaceprotocol.
-    // In NPM, Yarn, or PNPM if configured in such a way, this will be just the version.
-    expect(packageJson.devDependencies.sku).toEqual(
-      `workspace:${skuPackageJson.version}`,
-    );
-
```

Comment:
Ended up deleting this test because it stopped working. We can make it work by setting [`saveWorkspaceProtocol: true`](https://pnpm.io/9.x/npmrc#save-workspace-protocol), but I don't want to do that because in all other cases the default makes sense.

Happy for this to not be tested as it's relatively inconsequential because it's only relevant when running `sku init` with a snapshot version.